app/controllers/api/v1/reservations_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::V1::ReservationsController < ApplicationController
+  def index
+    user = current_user
+    @reservations = Reservation.includes(:tour).where(user_id: user.id)
+    render json: @reservations
```

Comment:
I suggest you add the tour data as part of the returned data. This will allow the frontend have access to the tour information for the respective reservations,

The endpoint currently returns:

```json
[
    {
        "id": 2,
        "start_end": "2023-10-20",
        "end_date": "2023-10-23",
        "user_id": 2,
        "tour_id": 1,
        "created_at": "2023-06-14T09:52:09.092Z",
        "updated_at": "2023-06-14T09:52:09.092Z"
    }
]
```

It should include tours data and return:

```json
[
    {
        "id": 2,
        "start_end": "2023-10-20",
        "end_date": "2023-10-23",
        "user_id": 2,
        "tour_id": 1,
        "created_at": "2023-06-14T09:52:09.092Z",
        "updated_at": "2023-06-14T09:52:09.092Z",
        "tour": {
            "id": 1,
            "name": "tour",
            "city": "lagos",
            "price": "1000.0",
            "video": "youtube.com",
            "status": null,
            "user_id": 1,
            "created_at": "2023-06-14T09:23:21.249Z",
            "updated_at": "2023-06-14T09:23:21.249Z"
        }
    }
]
```

I would suggest you also apply the above for the `show` endpoint